README.adoc

```diff
@@ -58,6 +55,10 @@ hello.html
 include::finish/src/main/webapp/hello.html[]
 ----
 
+== Additional prerequisites
+
+Before you begin, a Github account is required for the examples in this guide. 
+https://github.com/join[Register for a Github account^] if you do not already have one. 
```

Comment:
Suggested update

>Before you begin, a Github account is required for the examples in this guide. 
Register for a Github account if you do not already have one. 

To

>Before you begin, you must have a GitHub account to complete this guide. 
Register for a GitHub account, if you don't already have one.

Reply:
Update implemented